package.json

```diff
@@ -38,7 +38,7 @@
     "xmlhttprequest"
   ],
   "engines": {
-    "node": ">=4"
+    "node": ">=10"
```

Comment:
I'm not familiar enough with the `engines` entry in `package.json`, but my understanding is that it will require consumers to use Node 10 and above, otherwise they will receive warnings?

Since we are not using anything specifically from Node 10, I'm not sure if we should change this yet. Changing travis and the dockerfile to use Node 10 seems fine, though.

Reply:
The engine will prevent users from using the library if they are using older Node. 
The only issue I see with not updating `engine` is that if we are using latest ES standard it would fail on older Node because it won't be supported. 
To ensure that we might configure Travis CI to run tests on older Node also?

Reply:
As you suggested in a different PR, it might be good to configure Babel to transpile the result in that case.

While I agree it would be good to have consumers be on a newer version of Node, it feels unnecessary to make that a restriction when previously it wasn't, just so we can develop with the latest ES standard.